grunt/config.js

```diff
@@ -14,15 +14,12 @@ module.exports = {
   srcDir: 'src/',
   destDir: 'dist/',
 
-
-  requirejs: '../../node_modules/requirejs/require',
-
   // All files that should be checked with JSHint
   jsHintFiles: [
     'Gruntfile.js',
     'src/**/*.js',
-    'test/*.js',
-    'test/specs/**/*.js'
+    '!src/dist/**/*.js',
```

Comment:
TBH this doesn’t make sense to me at all. a dist folder in the src should be avoided at all cost IMO. But yes, I’m aware this is another issue not related to this PR.